homeassistant/components/onkyo/media_player.py

```diff
@@ -159,351 +149,191 @@ def service_handle(service):
         schema=ONKYO_SELECT_OUTPUT_SCHEMA,
     )
 
```

Comment:
The above needs to be `hass.services.async_register` since you are converting to async